tests/test_utils.py

```diff
@@ -205,6 +205,7 @@ def setUp(self):
             ignore_index=self.ignore_index,
         )
 
+    @unittest.skip("This test must be updated.")
```

Comment:
This test seems very related to the way the codellama/CodeLlama-7b-Instruct-hf tokenizer works (especially on BOS and EOS) and is not general. It should be rewritten in the future.